src/components/SQForm/SQFormReadOnlyField.js

```diff
@@ -12,6 +12,7 @@ function SQFormReadOnlyField({label, name, size = 'auto', muiFieldProps = {}}) {
   return (
     <Grid item sm={size}>
       <TextField
+        id={label.toLowerCase()}
```

Comment:
If a `label` is "First Name" is `first name` ok for the `id`?
Or do we need to hyphenate it? `first-name`

Reply:
Hmm, good question. I used `label.toLowerCase()` because that's what I saw was being used in other components, such as [`SQFormMultiSelect`](https://github.com/SelectQuoteLabs/SQForm/blob/master/src/components/SQForm/SQFormMultiSelect.js#L91).
I think a spaced string is technically valid, but something like kebab-case or camelCase would probably be better in terms of "best practices"